src/core/chemstate_mod.F90

```diff
@@ -1,4 +1,4 @@
-!> \file chemstate_mod.F90
+!>
```

Comment:
I think you still have to put `\file`(?), but not the file name itself, unless it's different from the file it's in

Reply:
I think I'm just going to add it back in.... we have it that way for all of the other files right now